fabtests/common/shared.c

```diff
@@ -2052,7 +2052,7 @@ ft_tag_is_valid(struct fid_cq * cq, struct fi_cq_err_entry *comp, uint64_t tag)
  * fi_cq_err_entry can be cast to any CQ entry format.
  */
 static int ft_spin_for_comp(struct fid_cq *cq, uint64_t *cur,
-			    uint64_t total, int timeout)
+			    uint64_t total, int timeout, int print_err)
```

Comment:
Instead of adding print_err, we can treat a timeout of 0 as meaning to drive progress.  If timeout is 0, we should skip the calls to clock_getime(), including the spot at the top of the function.